ui/desktop/tests/e2e/fixtures/authenticateTest.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef, no-empty-pattern */
+
+const { electronTest } = require('../fixtures/electronTest');
+const LoginPage = require('../pages/loginPage');
+
+exports.authenticateTest = electronTest.extend({
```

Comment:
C-L-E-A-N I really like it the usage of fixtures here!

Reply:
So this fixture is doing some basic setup before each test and authenticating for us correct?

Reply:
Yup, it's to simplify setup and not need to have `beforeAll/beforeEach` hooks for every test file